prow/cmd/sinker/main_test.go

```diff
@@ -373,6 +373,28 @@ func TestClean(t *testing.T) {
 				},
 			},
 		},
+		&corev1api.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "ttl-expired-while-still-pending",
```

Comment:
nit: It would be helpful to clarify that this is referring to a pending pod and that the ProwJob is completed. I was initially mislead by assuming that this meant the ProwJob was still pending.

Reply:
Done.